zerver/lib/events.py

```diff
@@ -111,6 +111,7 @@ def fetch_initial_state_data(
     slim_presence: bool = False,
     include_subscribers: bool = True,
     include_streams: bool = True,
+    spectator_requested_language: Optional[str] = None,
```

Comment:
Can we assert that this is not None in the spectator code path? I don't think that's a valid value for `default_language`.

Reply:
Okay, updated!